pages/security.md

```diff
@@ -190,6 +191,90 @@ The Okta developer blog also has some ❤️ for Micronaut and Quarkus:
 - [Build a Secure Micronaut and Angular App with JHipster](https://developer.okta.com/blog/2020/08/17/micronaut-jhipster-heroku)
 - [Fast Java Made Easy with Quarkus and JHipster](https://developer.okta.com/blog/2021/03/08/jhipster-quarkus-oidc)
 
+### Auth0
+
+If you'd like to use [Auth0](https://auth0.com/) instead of Keycloak, follow the configuration steps below:
+
+#### Create an OIDC App using Auth0 Admin Dashboard
+- Create a free developer account at <https://auth0.com/signup>. After successful sign-up, your account shall be associated with a unique domain like `dev-xxx.us.auth0.com`
+- Create a new application of type `Regular Web Applications`. Switch to the `Settings` tab, and configure your application settings like:
+    - Allowed Callback URLs: `http://localhost:8080/login/oauth2/code/oidc`
+    - Allowed Logout URLs: `http://localhost:8080/`
+    - NOTE: If you're using the JHipster Registry, add URLs for port 8761 too.
```

Comment:
BTW, it requires code upgrade on the jhipster registry side

Reply:
Good point. I [created an issue](https://github.com/jhipster/jhipster-registry/issues/511) to track.